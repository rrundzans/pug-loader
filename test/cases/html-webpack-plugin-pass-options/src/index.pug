h1 Hello world!
div= htmlWebpackPlugin.options.title