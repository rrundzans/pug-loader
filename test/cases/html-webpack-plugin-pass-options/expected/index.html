<h1>Hello world!</h1><div>My title</div>